release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -56,8 +56,65 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for Relational Expressions With All Ordered Types
+
+Relational expressions (`<`, `>`, `<=`, and `>=`) are supported with all [ordered types](https://ballerina.io/spec/lang/draft/v2020-12-17/#ordering). The static type of both operands must belong to the same ordered type.
+
+##### Support for inferring the `typedesc` argument of a dependently-typed function from the contextually expected type
+
+When the default value of a `typedesc` parameter of a dependently-typed function is `<>` and an argument is not provided for the parameter when calling the function, the argument will be inferred from the contextually expected type of the function call.
+```ballerina
+function func(typedesc<anydata> td = <>) returns td = external;
+
+public function main() {
+    // The argument for `td` is inferred to be `int`.
+    int value = func();
+}
+```
+
 #### Improvements
 
+##### Dependently-typed lang library functions that infer the argument from the contextually expected type
```

Comment:
```suggestion
##### Dependently-Typed Lang Library Functions that Infer the Argument from the Contextually-Expected Type
```

This is too long for a heading. Either we can rephrase it or make it a bullet point.

Reply:
Changed the capitalization, but as discussed offline, kept the heading as is since it's quite difficult to describe the feature by removing any of the current words.